locksmith/src/operations/stripeOperations.ts

```diff
@@ -135,54 +136,74 @@ export const disconnectStripe = async ({
 }
 
 /**
- * @deprecated
```

Comment:
I don't think this is deprecated @searchableguy ? It is the method used by our main flow to connect Stripe accounts...

Reply:
We have a v2 version. This is deprecated. Check v2 folder.

Reply:
smh, we have too many connectStripe

https://github.com/unlock-protocol/unlock/blob/173cb49f9bc07bbc15b854639061ba87a94a6d64/locksmith/src/controllers/lockController.ts#L10